samples/benchmarks/coremark/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
```

Consider the following

```suggestion

cmake_split_list(BOARD_PARTS "${BOARD}" "_")
list(GET BOARD_PARTS 0 BOARD_NAME)
list(GET BOARD_PARTS 1 SOC_NAME)
```

and then you could probably rsource in the following way:

```
rsource "boards/Kconfig.sysbuild_$SOC_NAME_soc_series"
```

---

That would cause Kconfig generation failures if Kconfig file for given SoC name does not exist. I would desist from doing it for now.

---

Also I think this CMakeLists.txt file executes after sysbuild

---

agree with Marek